source/elements/oneDAL/source/bibliography.rst

```diff
@@ -13,15 +13,19 @@ For more information about algorithms implemented in |dal_full_name|
    Available from `opencl-2.1.pdf <https://www.khronos.org/registry/OpenCL/specs/opencl-2.1.pdf>`_
 
 .. [SYCLSpec]
```

Comment:
> to double-check: do we follow the style suggested in ...

No. Why should we follow it?

> Also, in the generated docs I do not see [SYCLSpec] but a different tabbed way. Is this okay?

It's ok, because the way it's rendered in html depends on the theme. For example, in DAL docs we use the same theme as for spec, [the result is the following](http://oneapi-src.github.io/oneDAL/bibliography.html). Moreover, we use the [built-in RST directive](https://docutils.sourceforge.io/docs/user/rst/quickref.html#citations) for citations.

Reply:
On the first q: I think I shared Robert's input on it

@rscohn2, double-checking, if in https://build-me-the-docs-please.readthedocs.io/en/latest/Using_Sphinx/UsingBibTeXCitationsInSphinx.htm the style we want to follow in the oneAPI specs?

Reply:
@andreynv, as we agreed at the last meeting, we'll replace the current citation system with BibTeX. I'll create separate PR to address it for the entire oneDAL spec.